src/services/CoresService.Download.cs

```diff
@@ -197,7 +197,8 @@ public void DownloadCoreAssets(List<Core> cores)
 
         // These cores have instance json files and the roms are not in the default archive.
         // Check to see if they have a core specific archive defined, skip otherwise.
-        if (core.identifier is "Mazamars312.NeoGeo" or "Mazamars312.NeoGeo_Overdrive" or "obsidian.Vectrex"
+        // this is getting gross
+        if (core.identifier is "Mazamars312.NeoGeo" or "Mazamars312.NeoGeo_Overdrive" or "Mazamars312.NeoGeo_Analogizer" or "obsidian.Vectrex"

```

Comment:
I just saw the new core. I wonder if we should make this a JSON file to make these changes easier?

Reply:
yeah probably. a blacklist basically. I made this edit and release from my phone lol just to keep people from complaining